travis/test_server.py

```diff
@@ -142,8 +143,11 @@ def get_addons_path(travis_home, travis_build_dir, server_path):
     return addons_path
 
 
-def get_server_script(odoo_version):
-    return 'odoo-bin' if float(odoo_version) >= 10 else 'openerp-server'
+def get_server_script(server_path):
+    if os.path.isfile(os.path.join(server_path, 'odoo-bin')):
```

Comment:
Sorry! 
I know the try catch said but it is much easier!
Do not worry i'll fix it